genome_browser/_genome_diagram.py

```diff
@@ -88,25 +90,24 @@ def draw(self):
 
                 # Figure annotations will be applied to the last ax in offset
                 # coordinates in a lightgray text. Clipping is ignored as the
-                # text clearly cips with the axes outboard frame.
-                if self.annotation is not None:
-                    ax.annotate(
-                        xy=(1, 0),
-                        xycoords='axes fraction',
-                        s=self.annotation,
-                        xytext=(0, -60),
-                        textcoords='offset points',
-                        va='bottom',
-                        ha='right',
-                        color='0.6',
-                        clip_on=False)
+                # text clearly clips with the axes outboard frame.
+                ax.annotate(
+                    xy=(1, 0),
+                    xycoords='axes fraction',
+                    text=self.annotation,
+                    xytext=(0, -22.5),
```

Comment:
`-60` places the annotation outside my `ipython` window. The current version still looks fine to me:

![Screenshot from 2022-05-20 15-52-28](https://user-images.githubusercontent.com/13955209/169542678-06031ffe-8b9b-4ea8-98ac-93ba8c795d30.png)

I'm a `matplotlib` n00b, though, so open to suggestions. Main thing was to keep the demo working out of the box.

Reply:
BTW, updating the screenshot in `README.md` is probably overkill.